modules/microshift-nw-create-http-based-route.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-configuring-routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-nw-creating-a-route_{context}"]
+= Creating an HTTP-based route
+
+A route allows you to host your application at a public URL. It can either be secure or unsecured, depending on the network security configuration of your application. An HTTP-based route is an unsecured route that uses the basic HTTP routing protocol and exposes a service on an unsecured application port.
+
+The following procedure describes how to create a simple HTTP-based route to a web application, using the `hello-microshift` application as an example.
+
+.Prerequisites
+* You installed the OpenShift CLI (`oc`).
+* You have access to your {microshift-short} cluster.
+* You have a web application that exposes a port and a TCP endpoint listening for traffic on the port.
+
+.Procedure
+
+. Create a service called `hello-microshift` by running the following command:
++
+[source,terminal]
+----
+$ oc expose pod hello-microshift -n $namespace
+----
+
+. Create an unsecured route to the `hello-microshift` application by running the following command:
++
+[source,terminal]
+----
+$ oc expose svc/hello-microshift --hostname=microshift.com $namespace
+----
+
+.Verification
+
+* To verify that the `route` resource that you created, run the following command:
```

Comment:
Seems to keep the _<verb> by following_ pattern:
```suggestion
* Verify that the `route` resource was created by running the following command:
```
If not, I think you need something _To verify that the `route` resource that you created <does something>..._

```suggestion
* To verify that the `route` resource was created, run the following command:
```